converter/payload_converter.go

```diff
@@ -42,6 +42,11 @@ type PayloadConverter interface {
 	Encoding() string
 }
 
+type ProtoPayloadConverterInterface interface {
```

Comment:
I don't think this needs to be exported

Reply:
Ah, thanks. I was using capitals for using it outside of the file, but it's for outside of the package.